test/controllers/auth.test.js

```diff
@@ -65,31 +65,32 @@ describe(path, () => {
       expect(status).to.equal(200);
     });
 
-    it('should return a token', async () => {
-      const { body } = await request(server)
+    it('should have Set-Cookie header', async () => {
```

Comment:
I think this test should verify if there is a jwt in Set-Cookie header. Doesn't make sense to me to verify if only the header is present.

Reply:
The next test does that.
He just separated testing the existence of the header from testing the actual content.
Both are needed, in my opinion.